js-weather/assets/javascript/weather_fetcher.js

```diff
@@ -0,0 +1,52 @@
+export class WetherFetcher {
+  constructor() {
+    this.appId = "4b5774e9f3d2a07b84f0f2f88e486224";
+  }
+
+  async fetchByXMLHttpRequest(city) {
```

Comment:
(修正不要)
ここのasyncも無くてよいですね。


```suggestion
  fetchByXMLHttpRequest(city) {
```